peret/__init__.py

```diff
@@ -121,7 +117,7 @@ def _has_translation(e: TagNode, lang: str, value: str) -> bool:
     for quote in e.css_select(
         f'entry > sense > cit[type="translation"][xml|lang="{lang}"] > quote'
     ):
-        if value in map(str, quote.child_nodes()):
+        if value in quote.full_text:
```

Comment:
maybe this could also be a equality comparison.

Reply:
it probably should, come to think of it

Reply:
```suggestion
    return e.css_select(
        f'entry > sense > cit[type="translation"][xml|lang="{lang}"] > quote'
    ).filtered_by(
        lambda quote: quote.full_text == value
    ).size > 0
```

Reply:
is this a one-liner competition? but yeah.